### DISCUSSION TITLE
How can I find files that only have certain permission for owner?

### DISCUSSION THREAD

**Discussion by nghiepdev** [strapi/strapi#8701] [Legacy Topics]
Hi all,

Currently, anyone can find all files without restriction.

I think this a problem with security and policy.

So, How can I find files that only have certain permission for owner?

Thank you in advance.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
